Schema/models/LayerMetadata-StatusFields.d.ts

```diff
@@ -0,0 +1,16 @@
+import { LayerMetadata } from "./LayerMetadata";
```

Comment:
shouldn't this file name be in camel case - relevant for all the repo

Reply:
how " where the linter permits '

Reply:
Changed